package.json

```diff
@@ -9,13 +9,15 @@
   ],
   "scripts": {
     "build": "node_modules/.bin/babel src --out-dir dist --ignore test.js --source-maps --copy-files",
+    "coverage": "open coverage/lcov-report/index.html",
     "docs": "jsdoc2md --configure jsdoc.json src/**/*.js > API.md",
     "lint": "node_modules/.bin/eslint src test --fix",
-    "test": "npm run lint && node_modules/.bin/jest",
+    "test": "npm run lint && node_modules/.bin/jest src/*",
```

Comment:
Scripts use `node_modules/.bin` for pathing, so you should be able to `npm run lint && jest src/*`. I assume it works with multiple commands as well.